changelog.md

```diff
@@ -1,3 +1,9 @@
+2.17.2
+=========
+
+### New features
+- Added support for CIAM custom user domains. See issue [#2690](https://github.com/AzureAD/microsoft-identity-web/issues/2690) for details. 
```

Comment:
```suggestion
- Added support for CIAM custom user domains. You can now use an Open ID connect authority in the "Authority" property of the configuration instead of using "Instance" and "Tenant". See issue [#2690](https://github.com/AzureAD/microsoft-identity-web/issues/2690) for details. 
```